composer.json

```diff
@@ -31,6 +31,7 @@
         "symfony/dom-crawler": "^6.1",
         "silverstripe/testsession": "^3",
         "silverstripe/framework": "^5",
+        "silverstripe/supported-modules": "^1.1",
```

Comment:
Will already be included as part of framework, though I guess there's no harm explicitly including it here.